docs/v8-migration.md

```diff
@@ -0,0 +1,471 @@
+
+```diff title="babel.config.json"
+{
+  "plugins": ["@babel/transform-runtime", {
+-   corejs: 2
++   corejs: 3
+  }]
+}
+```
+
+### `@babel/plugin-syntax-import-assertions`
+Please migrate to `@babel/plugin-syntax-import-attributes` ([#15536](https://github.com/babel/babel/pull/15536)). After you replace the plugin, you should search and replace the following patterns in your codebase:
+
+```diff title="input.js"
+- import value from "module" assert { type: "json" };
++ import value from "module" with { type: "json" };
+```
+
+## Configuration Changes
+
+### `@babel/core` {#configuration-change-preset-core}
+
+![medium](https://img.shields.io/badge/risk%20of%20breakage%3F-medium-yellow.svg)
+
+- The root AMD/UMD/SystemJS options, namely [`moduleIds`](./options.md#moduleids), [`getModuleId`](./options.md#getmoduleid), [`moduleRoot`](./options.md#moduleroot), [`moduleId`](./options.md#moduleid) and [`filenameRelative`](./options.md#filenamerelative) are moved to plugin options ([#5473](https://github.com/babel/babel/issues/5473), [#12724](https://github.com/babel/babel/pull/12724)).
+
+  **Migration**: Move these options to the module plugin, for example, if you are using
+  `@babel/plugin-transform-modules-systemjs`:
+  ```js title="babel.config.js"
+  module.exports = {
+    plugins: [
+      ['@babel/plugin-transform-modules-systemjs', {
+        // highlight-start
+        moduleIds: true,
+        moduleRoot: 'myApp',
+        getModuleId (name) {
+          return name + "suffix";
+        },
+        // highlight-end
+      }],
+    ],
+  };
+  ```
+  Adapt the example above if you are using `@babel/plugin-transform-modules-amd` or `@babel/plugin-transform-modules-umd`. You can start the migration prior to Babel 8.0.
+
+  If you are using `@babel/cli` and passing `--module-ids`, `--module-root` and `--module-id` from command line, please create a Babel config `babel.config.js` and specify options there.
+
+### `@babel/preset-env` {#configuration-change-preset-env}
+
+![medium](https://img.shields.io/badge/risk%20of%20breakage%3F-medium-yellow.svg)
+
+- `includes` and `excludes` respect renamed package names ([#15576](https://github.com/babel/babel/pull/15576))
+
+  **Migration**: If `includes` or `excludes` contain any plugins mentioned in the [Packages Renames section](./v8-migration.md#package-renames), change it to the new name. For example,
+
+  ```diff title="babel.config.json"
+  {
+    "presets": [[
+      "@babel/preset-env",
+      {
+  -     "includes": ["proposal-optional-chaining"]
+  +     "includes": ["transform-optional-chaining"]
+      }
+    ]]
+  }
+  ```
+
+![low](https://img.shields.io/badge/risk%20of%20breakage%3F-low-yellowgreen.svg)
+
+- Remove `uglify` target ([#12594](https://github.com/babel/babel/pull/12594))
+
+  **Migration**: The `uglify` target had been deprecated since 7.0.0-beta.0, if you still need this, use the [`forceAllTransforms`](preset-env.md#forcealltransforms) option.
```

Comment:
```suggestion
  **Migration**: The `uglify` target had been deprecated since 7.0.0, if you still need this, use the [`forceAllTransforms`](preset-env.md#forcealltransforms) option.
```

Most people use only stable versions, no need to mention betas :)